src/throttle.decorator.ts

```diff
@@ -0,0 +1,47 @@
+import { THROTTLER_LIMIT, THROTTLER_SKIP, THROTTLER_TTL } from './throttler.constants';
+
+function setThrottlerMetadata(target: any, limit: number, ttl: number): void {
+  Reflect.defineMetadata(THROTTLER_TTL, ttl, target);
+  Reflect.defineMetadata(THROTTLER_LIMIT, limit, target);
+}
+
+/**
+ * Adds metadata to the target which will be handled by the ThrottlerGuard to
+ * handle incoming requests based on the given metadata.
+ * @usage @Throttle(2, 10)
```

Comment:
https://github.com/nestjs/docs.nestjs.com/pull/413  needs to have explicit information about what is public API. 

```suggestion
 * @usage @Throttle(2, 10)
 * @publicApi
```